dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -617,9 +618,9 @@
     <ItemGroup Condition="'$(MiniWindowsSDKWinMDPath)'!=''">
       <MetaDataDir Include="$(MiniWindowsSDKWinMDPath)" />
     </ItemGroup>
-    <Exec Command="$(MdMergeExe) -v @(MetaDataDir -> '-metadata_dir &quot;%(Identity)&quot;', ' ') -o &quot;$(UnmergedWinmdDirectory)\..&quot; -i &quot;$(UnmergedWinmdDirectory)&quot; -partial -n:3" />
-    <Exec Command="$(MdMergeExe) -v @(MetaDataDir -> '-metadata_dir &quot;%(Identity)&quot;', ' ') -o &quot;$(UnmergedWinmdDirectory)\..\sdk&quot; -i &quot;$(UnmergedWinmdDirectory)&quot; -partial -n:3 -createPublicMetadata -transformExperimental:transform" />
-    <Exec Command="$(MdMergeExe) -v @(MetaDataDir -> '-metadata_dir &quot;%(Identity)&quot;', ' ') -o &quot;$(MergedWinmdDirectory)&quot; -i &quot;$(UnmergedWinmdDirectory)&quot; -partial -n:2" />
+    <Exec Command="$(MdMergeExe) -v @(MetaDataDir -&gt; '-metadata_dir &quot;%(Identity)&quot;', ' ') -o &quot;$(UnmergedWinmdDirectory)\..&quot; -i &quot;$(UnmergedWinmdDirectory)&quot; -partial -n:3" />
```

Comment:
>-&gt [](start = 50, length = 4)

expect ->

Reply:
The CreateNewProject script is the one doing this, because it writes the project file back in proper xml. It is harmless.